packages/slate-editor/src/modules/components/FloatingSnippetInput/FloatingSnippetInput.module.scss

```diff
@@ -0,0 +1,15 @@
+@use "sass:math";
+@import "styles/variables";
+
+.FloatingSnippetInput {
+    $input-height: 34px;
+
+    position: relative;
+    top: math.div(-($input-height - $editor-paragraph-line-height), 2);
```

Comment:
I find `x * 0.5` much more readable than `math.div(x, 2)`.

Reply:
hm, as i understand it is used when there are not plain numbers
https://sass-lang.com/documentation/modules/math#div
i copy-pasted this part from other inputs, do not know what is right

Reply:
I believe it's producing the same result.

![image](https://user-images.githubusercontent.com/370680/191523549-b70604c1-fbbc-4478-a826-93ec4f9c4647.png)

Reply:
i tried to replace and got this warning 
<img width="816" alt="image" src="https://user-images.githubusercontent.com/3620639/191524579-064e71aa-065b-4d41-a821-33f55eb4b948.png">

Reply:
@aspirisen But you're using the deprecated `/` operator. I'm suggesting to use `* 0.5` instead of `Math.div(x, 2)`.

Reply:
hm, for me `* 0.5` is a bit implicit
however, if you insist i can use your suggestion